x/mongo/driver/topology/polling_srv_records_test.go

```diff
@@ -125,9 +125,20 @@ func compareHosts(t *testing.T, received []description.Server, expected []string
 }
 
 func TestPollingSRVRecordsSpec(t *testing.T) {
+	for _, uri := range []string{
+		"mongodb+srv://test1.test.build.10gen.cc/?heartbeatFrequencyMS=100",
+		"mongodb+srv://user:pass@test1.test.build.10gen.cc/?heartbeatFrequencyMS=100",
```

Comment:
```suggestion
                // Test with user:pass as a regression test for GODRIVER-2620
		"mongodb+srv://user:pass@test1.test.build.10gen.cc/?heartbeatFrequencyMS=100",
```

Since this is not a required specification test, suggest leaving a comment to note the motivation.